docs/testcoverage.rst

```diff
@@ -0,0 +1,86 @@
+Test Coverage
+====================
+
+You can integrate Coverage.py with behave-django to find out the test coverage of your code.
+
+Dependencies
+--------------------
+
+At First, you should install Coverage.py dependency
+
+.. code-block:: bash
+
+    $ pip install coverage
+
+Environment.py
+--------------------
+
+In environment.py , write below code on function before_all to start measuring test coverage
+
+.. code-block:: python
+
+    import coverage
+
+    def before_all(context):
+        cov = coverage.Coverage()
+        cov.start()
+        context.cov = cov
+
+Then write below code to end up measuring test coverage.
+You can save the coverage result on html format.
+
+.. code-block:: python
+
+    def after_all(context):
+        cov = context.cov
+        cov.stop()
+        cov.save()
+        cov.html_report(directory="./cov")
+
+
+You can check the test coverage on the web with the following command.
+
+.. code-block:: bash
+
+    $ python -m http.server --directory ./cov
+    
+Warning for behave-django
+--------------------
+
+Internally, the time before_all is executed seems to be later than the time when django loads the modules set in each app.
+
+So sometimes it is necessary to reload django app's modules for accurate test coverage measurement.
```

Comment:
Whoa! Reloading modules. That looks hacky. We should find a proper solution for that problem!

We have [django_ready](https://behave-django.readthedocs.io/en/stable/setup.html) already. The `before_all` should run before that, but it seems that [we actually give Django precedence](https://github.com/behave/behave-django/blob/master/behave_django/environment.py#L114-L115). That might need to be changed. A PR is welcome!

Reply:
Okay, I'll check the hook.